src/main.cpp

```diff
@@ -321,6 +322,24 @@ int main ( int argc, char** argv )
         }
 
 
+        // Quality of Service --------------------------------------------------
+        if ( GetNumericArgument ( argc,
+                                  argv,
+                                  i,
+                                  "-Q",
+                                  "--qos",
+                                  0,
+                                  255,
+                                  rDbleArgument ) )
+        {
+            iQosNumber            = static_cast<quint16> ( rDbleArgument );
+            qInfo() << qUtf8Printable( QString( "- selected tos value: %1" )
```

Comment:
Users aren't going to like "qos" for the parameter name and "tos" for the description of what it is when it's printed... :)